cms/djangoapps/contentstore/views/preview.py

```diff
@@ -214,7 +215,8 @@ def _prepare_runtime_for_preview(request, block):
         "teams_configuration": TeamsConfigurationService(),
         "sandbox": SandboxService(contentstore=contentstore, course_id=course_id),
         "cache": CacheService(cache),
-        'replace_urls': ReplaceURLService
+        'replace_urls': ReplaceURLService,
+        'video_config': VideoConfigService(course_id=course_id),
```

Comment:
Could you also refactor the existing video block to use this new service? I think that would make the extraction itself less risky, because the built-in and extracted blocks would share the same service code.